src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/SentenceCalculationService.kt

```diff
@@ -104,51 +101,6 @@ class SentenceCalculationService(private val sentenceAdjustedCalculationService:
     return DAYS.between(sentenceStartDate, date).toInt()
   }
 
-  private fun getTotalDuration(sentence: CalculableSentence): Duration {
-    return when (sentence) {
-      is StandardDeterminateSentence -> {
-        sentence.duration
-      }
-      is AFineSentence -> {
-        sentence.duration
-      }
-      is ExtendedDeterminateSentence -> {
-        sentence.combinedDuration()
-      }
-      is SopcSentence -> {
-        sentence.combinedDuration()
-      }
-      is SingleTermSentence -> {
-        sentence.combinedDuration()
-      }
-      else -> {
-        throw UnknownError("Unknown sentence in consecutive sentence")
-      }
-    }
-  }
-  private fun getCustodialDuration(sentence: CalculableSentence): Duration {
-    return when (sentence) {
-      is StandardDeterminateSentence -> {
-        sentence.duration
-      }
-      is AFineSentence -> {
-        sentence.duration
-      }
-      is ExtendedDeterminateSentence -> {
-        sentence.custodialDuration
-      }
-      is SopcSentence -> {
-        sentence.custodialDuration
-      }
-      is SingleTermSentence -> {
-        sentence.combinedDuration()
-      }
-      else -> {
-        throw UnknownError("Uknown sentence")
-      }
-    }
-  }
-
   private fun getSentenceCalculation(booking: Booking, sentence: CalculableSentence): SentenceCalculation {
```

Comment:
I know this isn't new, but the booking param never gets used in this method, can we remove it?